update/upgrade_db.sh

```diff
@@ -35,7 +27,7 @@ case $key in
     -h|--help)
         echo "Arguments:"
         echo -e "\t-c|--clean\t\tCleans comp and test DB before starting"
-        echo -e "\t-d|--localdirpath\tAdds local customization delta files"
+        echo -e "\t-d|--deltadirs\tAdds local customization delta files"
```

Comment:
Suggestion for the option description: "Add extra delta directories (for extensions/customizations)".